metrics/network/dockerfile_image/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#  This file is part of cc-oci-runtime.
```

Comment:
@jodh-intel you could have just said 'and in a number of other files, please grep and fix' :-)